tests/test_frameworks.py

```diff
@@ -40,14 +41,22 @@
 import skorch
 
 from cleanlab.classification import CleanLearning
-from cleanlab.experimental.keras import KerasWrapperSequential, KerasWrapperModel
+from cleanlab.models.keras import KerasWrapperSequential, KerasWrapperModel
+from cleanlab.internal.util import format_labels
 
 
 def python_version_ok():  # tensorflow and torch do not play nice with older Python
     version = sys.version_info
     return (version.major >= 3) and (version.minor >= 7)
 
 
+def run_fasttext_test():
+    # run test only if os enviroment is set of true and os is not Windows
+    if os.environ.get("TEST_FASTTEXT") == "true" and os.name != "nt":
+        return True
+    return False
```

Comment:
```suggestion
    return (os.environ.get("TEST_FASTTEXT") == "true" and os.name != "nt")
```

Reply:
You can omit the parentheses. That would be in line with changes in https://github.com/cleanlab/cleanlab/pull/600.

Reply:
thanks omitted